xchange-binance/src/main/java/info/bitrich/xchangestream/binance/BinanceStreamingExchange.java

```diff
@@ -67,7 +67,7 @@ private BinanceStreamingService createStreamingService(ProductSubscription subsc
     private static String buildSubscriptionStreams(ProductSubscription subscription) {
         return buildSubscriptionStrings(subscription.getTicker(), "ticker") +
                 buildSubscriptionStrings(subscription.getOrderBook(), "depth") +
-                buildSubscriptionStrings(subscription.getTrades(), "trade");
+                buildSubscriptionStrings(subscription.getTrades(), "aggTrade");
```

Comment:
the standard trade stream may still be implemented, there should be a new field for aggTrades